[2025-01-25T13:39:21.110+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-25T13:39:21.170+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: prim_pipelines.get_data_from_prim_api manual__2025-01-25T13:39:16.616714+00:00 [queued]>
[2025-01-25T13:39:21.189+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: prim_pipelines.get_data_from_prim_api manual__2025-01-25T13:39:16.616714+00:00 [queued]>
[2025-01-25T13:39:21.189+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-25T13:39:21.207+0000] {taskinstance.py:2889} INFO - Executing <Task(_PythonDecoratedOperator): get_data_from_prim_api> on 2025-01-25 13:39:16.616714+00:00
[2025-01-25T13:39:21.227+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=628) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-25T13:39:21.230+0000] {standard_task_runner.py:72} INFO - Started process 650 to run task
[2025-01-25T13:39:21.229+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'prim_pipelines', 'get_data_from_prim_api', 'manual__2025-01-25T13:39:16.616714+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpxwvos4kr']
[2025-01-25T13:39:21.231+0000] {standard_task_runner.py:105} INFO - Job 11: Subtask get_data_from_prim_api
[2025-01-25T13:39:21.314+0000] {task_command.py:467} INFO - Running <TaskInstance: prim_pipelines.get_data_from_prim_api manual__2025-01-25T13:39:16.616714+00:00 [running]> on host 51421c3b86a4
[2025-01-25T13:39:21.510+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='prim_pipelines' AIRFLOW_CTX_TASK_ID='get_data_from_prim_api' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T13:39:16.616714+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-25T13:39:16.616714+00:00'
[2025-01-25T13:39:21.512+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-25T13:39:22.491+0000] {python.py:240} INFO - Done. Returned value was: dags/data/data.json
[2025-01-25T13:39:22.544+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-25T13:39:22.545+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=prim_pipelines, task_id=get_data_from_prim_api, run_id=manual__2025-01-25T13:39:16.616714+00:00, execution_date=20250125T133916, start_date=20250125T133921, end_date=20250125T133922
[2025-01-25T13:39:22.586+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-25T13:39:22.639+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-01-25T13:39:22.641+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
